content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -78,8 +78,8 @@ HTTP verb for a **patch** is PATCH.
 
 ## Resource URIs
 
-All resource types are either scoped by the cluster (`/apis/GROUP/VERSION/*`) or to a
-namespace (`/apis/GROUP/VERSION/namespaces/NAMESPACE/*`). A namespace-scoped resource
+All resource types are either scoped by the cluster (`/api/GROUP/VERSION/*`) or to a
+namespace (`/api/GROUP/VERSION/namespaces/NAMESPACE/*`). A namespace-scoped resource
```

Comment:
`/apis/GROUP/VERSION` is correct for APIs placed in an API group.

Only the original "core" API group uses the `/api/v1/...` form